tutorials/detect.ipynb

```diff
@@ -87,6 +114,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adaptive smoothing\n",
+    "\n",
+    "For visualisation purpose it can be nice to look at a smoothed counts image. This can be performed using the adaptive smoothing algorithm from [Ebeling et al. (2006)](https://ui.adsabs.harvard.edu/abs/2006MNRAS.368...65E/abstract).\n",
+    "In the following example the `threshold` argument gives the minimum significance expected, values below are clipped.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "scales = u.Quantity(np.arange(0.05, 1, 0.05), unit=\"deg\")\n",
+    "smooth = ASmooth(threshold=3, scales=scales)\n",
+    "images = smooth.run(**maps)\n",
+    "\n",
+    "plt.figure(figsize=(15, 5))\n",
+    "images[\"counts\"].plot(add_cbar=True, vmax=10);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## TS map estimation\n",
+    "\n",
+    "The TS map is computed fitting by a single parameter amplitude fit as described in Appendix A of [Stewart (2009)](https://ui.adsabs.harvard.edu/abs/2009A%26A...495..989S/abstract). The fit is simplified by finding roots of the derivative of the fit statistics (default settings use [Brent's method](https://en.wikipedia.org/wiki/Brent%27s_method))."
```

Comment:
You might expand a bit here to have a more pedagogical tutorial.

Reply:
Added TS definition and reference